joplin/groups/fixtures/helpers/create_fixture.py

```diff
@@ -10,7 +10,7 @@ def create_fixture(group_data, fixture_name):
         group = None
     if group:
         print(f"Skipping {fixture_name}")
-        return None
+        return group
```

Comment:
Returning None was setting some of the groups in pages as Null, which then was violating a non-null constraint